### ISSUE TITLE
slamdunk snp(varscan2) missing SNP around INDEL

### ISSUE THREAD

**Issue by wzthhm** [t-neumann/slamdunk#117]
Hi,
I found slamdunk snp(varscan2) missing SNPs around INDEL(SNP-INDEL), and it seems to cause false positive, such as:

my mpileup file (use slamdunk snp argument create):

> chr4    5742730 T       73      C+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1G.C+1GC+1GC+1GC+1GC+1G.C+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC
+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1GC+1G
C+1GC+1G  FFBFFFFBFFF/FFFFF/FBFFFFF/FFFFF/F<<FFFFFFFFFF<BFFFFFFFFFFFFFFFFBF</BFFB</
but in .vcf file, "chr4    5742730" is missed

IGV:
<img width="1296" alt="1" src="https://user-images.githubusercontent.com/49448642/187694998-69384a88-aabe-4074-8816-ebe493e64385.png">

my command:
```
slamdunk map -r ../genome/mm10.fa -o ./ -t 160 ../data/SRR5678911.fastq
slamdunk filter -o ./ SRR5678911_slamdunk_mapped.bam
slamdunk snp -r ../genome/mm10.fa -o ./ -t 64 SRR5678911_slamdunk_mapped_filtered.bam
alleyoop read-separator -o ./ -s ./ -r ../genome/mm10.fa ./
```

I also check other "SNP-INDEL" locus, they have similar results. so I want to know what might be going wrong?
Thanks!

**Comment by t-neumann** [comment]
By default the variant fraction is from what I recall 0.8. Could be that this threshold is not met yet for these sites. For diploid samples we always set it to 0.2

**Comment by wzthhm** [comment]
Actually I don't know if "variant fraction" means "Count of observations of this alternate / All read counts", and if that's right, the variant fraction of "chr4 5742730" site should greater than 0.8 threshold. How do I deal with these sites?

**Comment by t-neumann** [comment]
Hm can u check in the resulting vcf file for this site, if there is anything reported?

**Comment by wzthhm** [comment]
Varscan2 doesn't report anything in the site..😭

**Comment by t-neumann** [comment]
Can you try running `slamdunk snp` with a very low variant fraction, like `-f 0.2` and see if then something is reported?

**Comment by wzthhm** [comment]
it doesn't work

**Comment by t-neumann** [comment]
Hm then I guess Varscan2 really chokes on the indel being right next to it. I would try to have a look into Varscan2 parameters that would allow for such calls

**Comment by wzthhm** [comment]
raw data: https://www.ncbi.nlm.nih.gov/sra?term=SRX2914406
slamdunk version: slamdunk 0.4.3

And actually I have tried bcftools and freebayes, both of them can't call this site.

**Comment by t-neumann** [comment]
Yeah this is unfortunately then a problem of the variant caller and not slamdunk itself. I couldnt really tune it to get it working. Do you have many such cases?
